cephfs/admin/mirror_workflow_test.go

```diff
@@ -25,6 +25,24 @@ const (
 	mirrorClient = "client.mirror_remote"
 )
 
+func waitForMirroring(t *testing.T, fsa *FSAdmin) {
+	for i := 0; i < 20; i++ {
+		modinfo, err := fsa.listModules()
+		require.NoError(t, err)
+		found := false
+		for _, emod := range modinfo.EnabledModules {
+			if emod == "mirroring" {
+				found = true
+				break
+			}
+		}
+		if found {
+			break
+		}
+		time.Sleep(10 * time.Millisecond)
+	}
```

Comment:
I think 10ms polling makes no sense. That also means you only poll for 200ms in total until timeout. And you can directly return from the inner loop. And shouldn't we fail, if mirroring is not enabled?

```suggestion
	for i := 0; i < 20; i++ {
		modinfo, err := fsa.listModules()
		require.NoError(t, err)
		for _, emod := range modinfo.EnabledModules {
			if emod == "mirroring" {
				return
			}
		}
		time.Sleep(100 * time.Millisecond)
	}
```